workflows/autometa_flagged.sh

```diff
@@ -0,0 +1,438 @@
+then
+    echo
+    echo "Coverage to be taken from Spades headers"
+    autometa-coverage --assembly $filtered_assembly --from-spades --out $coverages
+else
+    if [[ "$coverage" == "paired" ]]
+    then
+        echo
+        echo "Coverage to be calculated from reads"
+        if [ -z $fwd_reads ]; then
+            echo "Forward reads were not provided. These are required for coverage calculations"
+            exit 1
+        fi
+
+        if [ -f "$fwd_reads" ]; then
+            echo "$fwd_reads found."
+        else 
+            echo "$fwd_reads does not exist."
+            exit
+        fi
+
+        if [ -z $rev_reads ]; then
+            echo "Reverse reads were not provided. These are required for coverage calculations"
+            exit 1
+        fi
+
+        if [ -f "$rev_reads" ]; then
+            echo "$rev_reads found."
+        else 
+            echo "$rev_reads does not exist."
+            exit
+        fi
+
+        autometa-coverage --assembly $filtered_assembly --fwd-reads $fwd_reads --rev-reads $rev_reads --out $coverages --cpus $cpus
+
+    else
+        echo
+        echo "You have provided an invalid option. The options for coverage are 'spades' or 'paired'. Please fix">&2; exit 1
+    fi
+fi
+
+# Step 3: Annotate and filter markers
+
+orfs="${outdir}/${simple_name}.orfs.faa"
+orfs_nuc="${outdir}/${simple_name}.orfs.fna"
+
+#Get ORFs
+autometa-orfs \
+    --assembly $filtered_assembly \
+    --output-nucls $orfs_nuc \
+    --output-prots $orfs \
+    --cpus $cpus
+
+if [ $? -eq 0 ]; then
+   echo "ORFs successfully generated"
+else
+   echo "ORF generation failed"
+fi
+
+#Check if marker files are pressed, if not, press them
+echo
+echo "Checking if hmm files are pressed..."
+if [ -f "$marker_db/bacteria.single_copy.hmm.h3i" ]; then
+    echo "Bacterial hmm files are pressed, moving on..."
+else
+    echo "Bacterial hmm files not pressed. Pressing files now..."
+    hmmpress -f $marker_db/bacteria.single_copy.hmm
+fi
+
+
+if [ -f "$marker_db/archaea.single_copy.hmm.h3i" ]; then
+    echo "Archaeal hmm files are pressed, moving on..."
+else
+    echo "Archaeal hmm files not pressed. Pressing files now..."
+    hmmpress -f $marker_db/archaea.single_copy.hmm
+fi
+
+kingdoms=(bacteria archaea)
+
+# NOTE: We iterate through both sets of markers for binning both bacterial and archeal kingdoms
+for kingdom in ${kingdoms[@]};do
+    # kingdom-specific output:
+    hmmscan="${outdir}/${simple_name}.${kingdom}.hmmscan.tsv"
+    markers="${outdir}/${simple_name}.${kingdom}.markers.tsv"
+
+    # script:
+    autometa-markers \
+        --orfs $orfs \
+        --hmmscan $hmmscan \
+        --dbdir $marker_db \
+        --out $markers \
+        --kingdom $kingdom \
+        --parallel \
+        --cpus $cpus \
+        --seed $seed
+done
+
+# Step 4.1: Determine ORF lowest common ancestor (LCA) amongst top hits
+
+#Run blastp
+blast="${outdir}/${simple_name}.blastp.tsv" #Generate output file name
+
```

Comment:
See comment for `autometa-markers` above

```suggestion
set -x
```